readthedocs/settings/base.py

```diff
@@ -754,6 +754,11 @@ def DOCKER_LIMITS(self):
     # Stripe
     # Existing values we use
     STRIPE_SECRET = None
+    # NOTE: Remove this when enabling djstripe,
+    # since isn't recommended to set this value,
+    # we should always use the default version
+    # that djstripe works with.
+    STRIPE_VERSION = "2020-08-27"
```

Comment:
I'd like to see a similar note next to the djstripe package dependency in the `requirements.txt` file. I'd say it should hard pin to a specific version with a note like this explaining what are the required actions when upgrading djstripe. Otherwise, we will hit this issue again and break our subscriptions in the future.